seminar/feedpage/views.py

```diff
@@ -28,7 +28,7 @@ def show(request, id):
     if request.method == 'POST':
         feed.title = request.POST['title']
         feed.content = request.POST['content']
-        feed.update_date()
+        Feed.objects.filter(id=id).update(title=feed.title, content=feed.content)
```

Comment:
넵 원래 의도했던 코드는
```suggestion
        Feed.objects.update(id=id,title=feed.title, content=feed.content)
```
입니다. 다음 과제할 때 반영해보세요~